packages/blocks/src/database-block/database-model.ts

```diff
@@ -180,9 +228,8 @@ export const DatabaseBlockSchema = defineBlockSchema({
   flavour: 'affine:database',
   props: (internal): Props => ({
     title: internal.Text(),
-    columns: [],
     yCells: internal.Map<Y.Map<unknown>>(),
```

Comment:
Is it possible renaming `yCells` -> `cells`?

Reply:
Yeah, and we'll have an object proxy for it.